lib/bloc/pos_catalog/pos_csv_utils.dart

```diff
@@ -0,0 +1,108 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:breez/bloc/pos_catalog/model.dart';
+import 'package:breez/logger.dart';
+import 'package:csv/csv.dart';
+import 'package:flutter/foundation.dart';
+import 'package:path_provider/path_provider.dart';
+
+class PosCsvUtils {
+  final List itemList;
+
+  PosCsvUtils({this.itemList});
+
+  Future export() async {
+    log.info("export pos items started");
+    String tmpFilePath =
+        await _saveCsvFile(const ListToCsvConverter().convert(_generateList()));
+    log.info("export pos items finished");
+    return tmpFilePath;
+  }
+
+  List _generateList() {
+    log.info("generating payment list started");
+    List<List<dynamic>> paymentList =
+        List.generate(this.itemList.length, (index) {
+      List paymentItem = List();
+      Item paymentInfo = this.itemList.elementAt(index);
+      paymentItem.add(paymentInfo.id.toString());
+      paymentItem.add(paymentInfo.name);
+      paymentItem.add(paymentInfo.sku);
+      paymentItem.add(paymentInfo.imageURL);
```

Comment:
Currently null values will be printed as 'null' in the CSV. Isn't it better to print empty strings instead?